src/components/Modal.js

```diff
@@ -15,7 +17,11 @@ class Modal extends Component {
 
   scrollContentsToTop() {
     if (this.contentRef.current) {
-      this.contentRef.current.scrollTop = 0;
+      animejs({
+        targets: this.contentRef.current,
+        scrollTop: 0,
+        duration: getCSSVariableAsNumber('--animation-duration-slow-ms'),
```

Comment:
Adding an `easing: getCSSVariableAsObject('--animation-easing-js'),` removes the 'bounce' that animejs defaults to.

Reply:
Thank you — fixed.